src/types/annotation.ts

```diff
@@ -14,14 +14,14 @@ export interface AnnotationStream {
 
 export interface GetAnnotationPayload {
   stream?: string
-  start?: string
-  end?: string
+  start?: number
+  end?: number
   stickers?: any
 }
 
 export interface DeleteAnnotation {
   stream: string
-  start: string
-  end: string
+  start: number
+  end: number
```

Comment:
you'll want to change these back, they don't have any bearing on this problem, and they were correct before

Reply:
For sure!